how-to/integrate-with-bloomberg-basic/public/manifest.fin.json

```diff
@@ -0,0 +1,57 @@
+{
+	"devtools_port": 9090,
+	"licenseKey": "openfin-demo-license-key",
+	"runtime": {
+		"arguments": "--v=1 --inspect",
+		"version": "29.108.73.14"
```

Comment:
Please update the version to runtime 30 to match the runtime used by workspace-platform v12.